Kitodo/src/main/java/org/kitodo/production/forms/CommentForm.java

```diff
@@ -0,0 +1,281 @@
+/*
+ * (c) Kitodo. Key to digital objects e. V. <contact@kitodo.org>
+ *
+ * This file is part of the Kitodo project.
+ *
+ * It is licensed under GNU General Public License version 3 or later.
+ *
+ * For the full copyright and license information, please read the
+ * GPL3-License.txt file that was distributed with this source code.
+ */
+
+package org.kitodo.production.forms;
+
+import java.util.Date;
+import java.util.List;
+import java.util.Objects;
+
+import javax.enterprise.context.SessionScoped;
+import javax.inject.Named;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.kitodo.data.database.beans.Comment;
+import org.kitodo.data.database.beans.Process;
+import org.kitodo.data.database.beans.Task;
+import org.kitodo.data.database.enums.CommentType;
+import org.kitodo.data.database.exceptions.DAOException;
+import org.kitodo.data.exceptions.DataException;
+import org.kitodo.production.enums.ObjectType;
+import org.kitodo.production.helper.Helper;
+import org.kitodo.production.helper.WikiFieldHelper;
+import org.kitodo.production.helper.batch.BatchTaskHelper;
+import org.kitodo.production.services.ServiceManager;
+
+@Named("CommentForm")
+@SessionScoped
+public class CommentForm extends BaseForm {
+    private static final Logger logger = LogManager.getLogger(CommentForm.class);
+    private boolean correctionComment = false;
+    private String commentMessage;
+    private String correctionTaskId;
+    private String processId;
+    private Process process;
+    private BatchTaskHelper batchHelper;
+
+
+    public List<Comment> getAllComments() {
+        WikiFieldHelper.transformWikiFieldToComment(this.process);
```

Comment:
I would condition this with an nonEmpty wikifield.

Reply:
This test will be done in "transformWikiFieldToComment()" method.